ex8.1_hierarchy.py

```diff
@@ -0,0 +1,87 @@
+class RegularUser:
+    """
+    managing the regular users
```

Comment:
This class doesnt "manage" anything, fix to proper english and proper documentation